unittest/commands/CMakeLists.txt

```diff
@@ -2,16 +2,25 @@
 
 add_executable(test_simple_commands test_simple_commands.cpp)
 
-# tests for the plugin command require the PLUGIN package
-if(PKG_PLUGIN)
+# tests for the plugin command require the PLUGIN package and won't work on windows
+if((NOT (CMAKE_SYSTEM_NAME STREQUAL "Windows")) AND PKG_PLUGIN)
```

Comment:
@akohlmey why did you re-add this? It won't build the plugins in that case.

Reply:
Not intentionally.

Reply:
That would explain why it works on my virtual machine and not on GitHub: it uses the old plugins, but those are missing with the GitHub runner.